src/components/SidebarItem/SidebarItem.jsx

```diff
@@ -48,7 +48,8 @@ export default class SidebarItem extends React.Component {
         <Link
           key={this.props.url}
           className={`${block}__title`}
-          to={this.props.url}>
+          to={this.props.url}
+          onClick={()=>window.scrollTo(0,0)}>
```

Comment:
Can you refactor this callback into a function outside `render` or even Component?

E.g.,

```js
function scrollTop() {
  window.scrollTo(0, 0)
}
```
Then we use it in `onClick`:

```js
onClick={scrollTop}
```

Reply:
BTW, do you have idea what might caused that issue?

Reply:
do you really need that way, because that really don't make seance, as it's not required anywhere else if you still need that way i can push another commit with the changes, or you can just merge this one

And about the cause, it just happen due to not use of '#' at the end of url **but** that even don't work as we use  `react-router-dom` for the link which by default prevent the link default behavior to execute

Reply:
Indeed, it's not required, but I think it's a [best practice](https://reactjs.org/docs/faq-functions.html#arrow-function-in-render) in React so it doesn't hurt to refactor it, what do you think?

Reply:
well, it depends, since the change needed is minute(very little), so i don't think it's needed to include extra lines unnecessarily right now.  but if you finally needed then i will push another commit with the changes. what you finally suggest

Reply:
well i have made the changes as required...!